src/backy/utils.py

```diff
@@ -359,14 +359,13 @@ def files_are_equal(a, b):
         chunk_a = a.read(CHUNK_SIZE)
         chunk_b = b.read(CHUNK_SIZE)
         if chunk_a != chunk_b:
-            logger.error(
-                "Chunk A ({}, {}) != Chunk B ({}, {}) at position {}".format(
-                    repr(chunk_a),
-                    hashlib.md5(chunk_a).hexdigest(),
-                    repr(chunk_b),
-                    hashlib.md5(chunk_b).hexdigest(),
-                    position,
-                )
+            log.error(
```

Comment:
yeah, printing the chunk isn't a good idea.  quarantining would be a better idea.